docs/usage/nfd-export.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Export"
+layout: default
+sort: 12
+---
+
+# Feature Export
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+## Export
+
+If you are interested in exporting features or labels in a generic
+context, the nfd client supports an export mode, where both can be
+derived on the command line.
+
+### Features
+
+**Feature export is in the experimental `v1alpha1` version.**
```

Comment:
Is it OK to refer to this as `v1alpha1` even if isn't an API but a command line option?

Reply:
We could just say it is an experimental version. Would that be better?

Reply:
Done!